<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolverResultsGraphicWidget</class>
 <widget class="QWidget" name="SolverResultsGraphicWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QFrame" name="graphic_frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="0">
          <widget class="QLabel" name="min1_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="colorbar1">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLineEdit" name="colorbar2">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3" colspan="3">
          <widget class="cinolib::GLcanvasOld" name="canvas2" native="true"/>
         </item>
         <item row="0" column="3" colspan="3">
          <widget class="QComboBox" name="canvas2_cb">
           <property name="currentText">
            <string>Solver Output (V)</string>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Solver Output (V)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GroundTruth (V)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>H1 Error (P)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>L2 Error (P)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Condition Number (P)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QComboBox" name="canvas1_cb">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>Solver Output (V)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GroundTruth (V)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>H1 Error (P)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>L2 Error (P)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Condition Number (P)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="cinolib::GLcanvasOld" name="canvas1" native="true"/>
         </item>
         <item row="2" column="5">
          <widget class="QLabel" name="max2_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="min2_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="max1_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="6">
          <widget class="QSlider" name="t_slider">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cinolib::GLcanvasOld</class>
   <extends>QWidget</extends>
   <header location="global">gui-opengl/glcanvas_old.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
